sorc/link_fv3gfs.sh

```diff
@@ -227,11 +227,18 @@ for ufs_utilsexe in \
 done
```

Comment:
The paths for the DA Monitoring files need to be updated.  Since release/gfsda.v16.3.0 no longer includes version numbers in the Monitoring packages, the .v1.0.0, .v1.0.1, .v2.0.0, and .v3.0.0 parts of gdas.v1.0.0, gfs.v1.0.0, minmon_shared.v1.0.1, gdas_oznmon.v2.0.0, oznmon_shared.v2.0.0, gdas_radmon.v3.0.0, and radmon_shared.v3.0.0 will need to be removed.  These changes will affect lines 157 - 193.

Reply:
done.